tests/test_utils.py

```diff
@@ -198,3 +202,19 @@ def test_letter_seq():
     ints = aa_seq_to_int(seq)
     one_hot = np.stack([one_hots[i] for i in ints])
     assert letter_seq(one_hot) == seq
+
+
+@given(st.data())
+@settings(deadline=None, max_examples=20)
+def test_seq_to_oh(data):
```

Comment:
Let's get a docstring on this test, to ensure that the docstring coverage, even on tests, remains high.